src/partition/repart.c

```diff
@@ -479,6 +480,43 @@ static uint64_t round_up_size(uint64_t v, uint64_t p) {
         return v * p;
 }
 
+/* calculates the size of a dm-verity hash partition's contents */
+static int calculate_verity_hash_size(
+                uint64_t *ret_bytes,
+                uint64_t data_bytes,
+                uint64_t hash_block_size,
+                uint64_t data_block_size) {
+        uint8_t digest_size = SHA256_DIGEST_SIZE;
```

Comment:
Please insert an empty line between function arguments and variable declarations.

BTW, is it necessary to declare this? Why not to use `SHA256_DIGEST_SIZE` directly in below?

Reply:
No, I don't think it is. I originally started with the digest size as a function parameter and then realized that it is essentially a hard-coded value in repart, so I moved to that assignment.